client/app/components/ovh-angular-sso-auth-modal-plugin/ovh-angular-sso-auth-modal-plugin.module.js

```diff
@@ -0,0 +1,3 @@
+angular.module("App").config((ssoAuthModalPluginFctProvider) => {
+    ssoAuthModalPluginFctProvider.setTranslationsPath("node_modules/ovh-angular-sso-auth-modal-plugin/dist/modal");
```

Comment:
will the path be ok in the prod dist ?

Reply:
Good question :-)
I will tell you when it will be on test env ;-)

Reply:
i think you'll have to update Assets.js too